pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -8,69 +8,143 @@
 permalink: /eks-for-enterprise/pricing/
 
 categories:
-  - title: Feature
+  - title: EKS
+    features:
+      - name: Control plane
+        description: Support for configuring, deploying, and managing the EKS control plane.
+        plans: ["Standard", "Enterprise"]
+      - name: Worker nodes
+        description: "Support for deploying any of the three worker node types: self-managed ASG, EKS Managed Node Groups, and Fargate."
+        plans: ["Standard", "Enterprise"]
+      - name: Core services
+        description: Suite of core Kubernetes applications every EKS cluster should have, including AWS LoadBalancer controller, external-dns, cluster-autoscaler, and more.
+        plans: ["Standard", "Enterprise"]
+      - name: CI/CD
+        description: Support for setting up an infrastructure CI/CD pipeline using traditional CI services (e.g., Jenkins, CircleCI, GitLab CI, etc).
+        plans: ["Standard", "Enterprise"]
+      - name: Basic monitoring
+        description: Support for metric and log aggregation to AWS CloudWatch.
+        plans: ["Standard", "Enterprise"]
+      - name: Basic deployment models
+        description: Support for rolling deployments where a configurable number of service Pods are replaced at a time.
+        plans: ["Standard", "Enterprise"]
+      - name: Basic networking
+        description: Support for AWS VPC based networking for Kubernetes Pods.
+        plans: ["Standard", "Enterprise"]
+      - name: Cluster auto scaling
+        description: Support for EKS worker node auto scaling throught the Kubernetes cluster-autoscaler.
+        plans: ["Standard", "Enterprise"]
+      - name: K8s native CI/CD
+        description: Support for CI/CD workflows using Kubernetes native services such as Argo CD and Flux.
+        plans: ["Enterprise"]
+      - name: Advanced monitoring
+        description: Support for metric and log aggregation through various 3rd party and in-cluster options, such as DataDog, Prometheus, Grafana, Elastic Stack, etc.
+        plans: ["Enterprise"]
+      - name: Advanced deployment models
+        description: Support for wide range of deployment models including canary, blue/green, automatic rollbacks, etc.
+        plans: ["Enterprise"]
+      - name: Advanced networking
+        description: Support for wide range of Pod networking options including Calico and Service Mesh.
+        plans: ["Enterprise"]
+      - name: Pod auto scaling
```

Comment:
Advanced auto scaling

Reply:
I thought about the alternative way, but then realized that it's not as clean a split. E.g., the monitoring and networking options actually apply to both, as you get both cluster level logs + metrics and pod level. Similarly, service mesh arguably is service networking.

I think it's better to keep the table simple like it is now.